simulator/mips/mips_instr.h

```diff
@@ -326,12 +326,14 @@ class FuncInstr
         uint32 lo = NO_VAL32;
 
         FuncInstr() = delete; // constructor w/o arguments for ports
+ 
+        FuncInstr( const FuncInstr& instr) = default;
```

Comment:
We tend to use "big zero" rule, which is opposite to "big five" rule: if none of the five special methods are implemented, do not implement all others (even with `=default`):
* destructor
* copy constructor
* move constructor
* copy assignment
* move assignment